boring/src/ssl/mod.rs

```diff
@@ -696,16 +696,20 @@ pub struct SslContextBuilder {
 impl SslContextBuilder {
     /// Creates a new `SslContextBuilder` to be used with Raw Public Key.
     ///
-    /// This corresponds to [`SSL_CTX_new`].
+    /// This corresponds to calling [`SSL_CTX_new`] then setting the default key exchange
+    /// preferences.
```

Comment:
Worth updating code comments with information that default curves will be changed based on the featureset

Reply:
Yup! Done.